exec/java-exec/src/main/resources/drill-module.conf

```diff
@@ -189,6 +189,15 @@ drill.exec: {
   debug: {
     return_error_for_failure_in_cancelled_fragments: false
   }
+  udf: {
+    retry.times: 5,
+    directory: {
+      base: "/tmp",
+      staging: ${drill.exec.zk.root}"/udf/staging",
```

Comment:
I wonder, are we trying to define two distinct things here? A ZK zNode and a DFS directory? If so, perhaps split them between

drill.exc.udf.dynamic
  dfs
    directory props...
  zk
     zk zNodes

It is not (yet) clear exactly how these are used, so this comment may be missing the point if we configure only dfs locations...